pkgs/servers/nostr-rs-relay/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, openssl
+, pkg-config
+, libiconv
+, darwin
+, protobuf
+,
+}:
+rustPlatform.buildRustPackage rec {
```

Comment:
```suggestion
}:

rustPlatform.buildRustPackage rec {
```

Reply:
thanks, I missed the message detail, makes sense.